src/SvgWrapper.js

```diff
@@ -442,159 +488,181 @@ class SvgWrapper {
    * @param {Number} attachedPseudoElement.hyrogenCount The number of hydrogens attached to each atom matching the key.
    */
   drawText(x, y, elementName, hydrogens, direction, isTerminal, charge, isotope, attachedPseudoElement = {}) {
-    let offsetX = this.offsetX,
-      offsetY = this.offsetY,
-      pos = {
-        x: x + offsetX,
-        y: y + offsetY,
-      },
-      textElem = document.createElementNS('http://www.w3.org/2000/svg', 'text'),
-      writingMode = 'horizontal-tb',
-      letterSpacing = 'normal',
-      textOrientation = 'mixed',
-      textDirection = 'direction: ltr;',
-      xShift = -2,
-      yShift = 2.5;
+    let text = [];
+    let display = elementName;
 
-    let mask = document.createElementNS('http://www.w3.org/2000/svg', 'circle');
-    mask.setAttributeNS(null, 'cx', pos.x);
-    mask.setAttributeNS(null, 'cy', pos.y);
-    mask.setAttributeNS(null, 'r', '3.5');
-    mask.setAttributeNS(null, 'fill', 'black');
-    this.maskElements.push(mask);
-
-    // determine writing mode
-    if (/up|down/.test(direction) && !isTerminal) {
-      writingMode = 'vertical-rl';
-      textOrientation = 'upright';
-      letterSpacing = '-1px';
-    }
-
-    if (direction === 'down' && !isTerminal) {
-      xShift = 0;
-      yShift = -2;
-    } else if (direction === 'up' && !isTerminal) {
-      xShift = 0.5;
-    } else if (direction === 'left') {
-      xShift = 2;
-    }
-
-    if (direction === 'left' || (direction === 'up' && !isTerminal)) {
-      textDirection = 'direction: rtl; unicode-bidi: bidi-override;'
-    }
-
-    // now the text element
-    textElem.setAttributeNS(null, 'x', pos.x + xShift);
-    textElem.setAttributeNS(null, 'y', pos.y + yShift);
-    textElem.setAttributeNS(null, 'class', 'element');
-    textElem.setAttributeNS(null, 'fill', this.themeManager.getColor(elementName));
-    textElem.setAttributeNS(null, 'style', `
-                text-anchor: start;
-                writing-mode: ${writingMode};
-                text-orientation: ${textOrientation};
-                letter-spacing: ${letterSpacing};
-                ${textDirection}
-            `);
-
-    let textNode = document.createElementNS('http://www.w3.org/2000/svg', 'tspan');
-    // special case for element names that are 2 letters
-    if (elementName.length > 1) {
-      let textAnchor = /up|down/.test(direction) ? 'middle' : 'start';
-
-      textNode.setAttributeNS(null, 'style', `

```

Comment:
Getting rid of this css based styling was a good call. It was causing problems with printing outside the browser.